app/views/users/show.html.erb

```diff
@@ -10,5 +10,6 @@
         </li>
     <% end %>
     <br/>
+    <br/>
     <%= link_to 'show more', user_posts_path(@user.id) %>
```

Comment:
- [ ] Kindly change the text to `See all Post` as we have it on the wireframe

![see-all-post](https://github.com/BenM4k/BlogApp/assets/26597432/8b8f24e1-33b7-4487-a13a-231d4195daf1)